hydromt_wflow/wflow.py

```diff
@@ -987,6 +1052,50 @@ def setup_laimaps(self, lai_fn: Union[str, xr.DataArray]):
         rmdict = {da_lai.dims[0]: "time"}
         self.set_grid(da_lai.rename(rmdict), name="LAI")
 
+    def setup_laimaps_from_lulc_mapping(
+        self,
+        lulc_fn: Union[str, xr.DataArray],
+        lai_mapping_fn: Union[str, pd.DataFrame],
+    ):
+        """
+        Derive cyclic LAI maps based on a LULC-LAI mapping table.
+
+        Adds model layers:
+
+        * **LAI** map: Leaf Area Index climatology [-]
+            Resampled from source data using average. Assuming that missing values
+            correspond to bare soil, these are set to zero before resampling.
+
+        Parameters
+        ----------
+        lulc_fn : str, xarray.DataArray
+            Name of RasterDataset source for landuse-landcover data.
+        lai_mapping_fn : str, pd.DataFrame
+            Path to a mapping csv file from landuse in source name to
+            LAI values. The csv file should contain lines with landuse classes
+            and LAI values for each month. The columns should be named as the
+            months (1,2,3,...,12).
```

Comment:
I can say that it can but it can also be your own data like the landuse mapping table for other paramters. I will for example publish global tables based on pfafstetter 2 on the p drive so then you actually wouldn't need to run setup_laimaps if you want to use these.